CRM/Contribute/BAO/Contribution.php

```diff
@@ -3691,7 +3691,10 @@ public static function recordFinancialAccounts(&$params, $financialTrxnValues =
           $params['prevContribution']->contribution_status_id != $params['contribution']->contribution_status_id
         ) {
           //Update Financial Records
-          self::updateFinancialAccounts($params, 'changedStatus');
+          $callUpdateFinancialAccounts = self::updateFinancialAccountsOnContributionStatusChange($params);
```

Comment:
this var isn't strictly needed but I used it because the function being called does not do a good job of implying the meaning of it's return value